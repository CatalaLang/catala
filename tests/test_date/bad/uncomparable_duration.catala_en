## `UncomparableDurations` exception management

### `<^` operator

```catala
declaration scope Lt:
  context d content boolean

scope Lt:
  definition d equals 1 month <^ 2 day
```

### `<=^` operator

```catala
declaration scope Le:
  context d content boolean

scope Le:
  definition d equals 1 month <=^ 2 day
```

### `<=^` operator

```catala
declaration scope Gt:
  context d content boolean

scope Gt:
  definition d equals 1 month >^ 2 day
```

### `>=^` operator

```catala
declaration scope Ge:
  context d content boolean

scope Ge:
  definition d equals 1 month >=^ 2 day
```
