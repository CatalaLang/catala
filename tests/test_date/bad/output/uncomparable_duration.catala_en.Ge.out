[ERROR] Cannot compare together durations that cannot be converted to a precise number of days
[ERROR] 
[ERROR]   --> test_date/bad/uncomparable_duration.catala_en
[ERROR]    | 
[ERROR] 40 |   definition d equals 1 month >=^ 2 day
[ERROR]    |                       ^^^^^^^
[ERROR]    + `UncomparableDurations` exception management
[ERROR]    +-+ `>=^` operator
[ERROR] 
[ERROR]   --> test_date/bad/uncomparable_duration.catala_en
[ERROR]    | 
[ERROR] 40 |   definition d equals 1 month >=^ 2 day
[ERROR]    |                                   ^^^^^
[ERROR]    + `UncomparableDurations` exception management
[ERROR]    +-+ `>=^` operator
